app/components/Checkbox/index.js

```diff
@@ -34,14 +34,22 @@ const Checkbox = ({ children }) => (
         color="#333536"
         letterSpacing="0.13px"
       >
-        {children}
+        {label}
       </Box>
     </label>
   </Box>
 );
 
 Checkbox.propTypes = {
-  children: PropTypes.string,
+  label: PropTypes.string,
+  iconName: PropTypes.string,
+  iconSize: PropTypes.string,
+};
+
+Checkbox.defaultProps = {
+  iconName: 'customCheck',
+  iconSize: '10px',
+  label: 'default label',
```

Comment:
Why are we exposing the proptypes for these base styles. We are not going to expose these props,
This is why before making a commit, we all need to agree on making a change to our codebase.
We need the children to be exposed outside,
since our checkbox is used like this `<Checkbox>Three</Checkbox>`

Reply:
Kindly refer how Chakra has built the Checkbox label, it should be only children
https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/Checkbox/index.js

Icon Name and Icon Size is going to be the same, if we are going to expose that prop that should have gone through the discussion in the RFC.

We have already started the conversion, we wont be exposing icon name, because throughout the project, the same name will be used.

We can expose the iconsize but you can mention that in the RFC.